<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewScape</class>
 <widget class="QMainWindow" name="ViewScape">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViewScape</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="videoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="reproductionSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>761</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="soundSlider">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>520</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timeProgressLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>490</y>
      <width>81</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00 / 00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>520</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>OPEN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="volumeButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>520</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="expandButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>520</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>520</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathFileEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
